_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -38,9 +38,7 @@ This rule applies to any element that has:
 
 ## Expectation
 
-The complete [visible text content][] of the target element either matches or is contained within its [accessible name][].
-
-**Note:** Leading and trailing [whitespace][] and difference in case sensitivity should be ignored.
+All [text nodes][] in the [visible text content][] of each target element matches, or is contained within the [accessible name][] of the target element, except for [text nodes][] contains [non-text content][]. Leading and trailing [whitespace][] and difference in case sensitivity should be ignored.
```

Comment:
```suggestion
Each [text node][] in the [visible text content][] of each target element matches, or is contained within, the [accessible name][] of this target element, except for [text nodes][] containing [non-text content][]. Leading and trailing [whitespace][] and difference in case sensitivity should be ignored.
```
Maybe even un-nesting a bit to avoid the "except":
```suggestion
Each [text node][] that is in the [visible text content][] of each target element and does not contain [non-text content][] matches, or is contained within, the [accessible name][] of the target element. Leading and trailing [whitespace][] and difference in case sensitivity should be ignored.
```

Reply:
I think it's better to use the word "except" to be clear something is an exception, rather then use "or" to chain two conditions together.